### ISSUE TITLE
Won't run after git pull origin main

### ISSUE THREAD

**Issue by chrism** [uwe-app/website#2]
Trying to get the latest version to read the integrations docs...

```
> git pull origin main
> uwe --live\
INFO  workspace::options > clean /Users/chrismasters/git/uwe-website/build/debug
INFO  workspace::options > mkdir /Users/chrismasters/git/uwe-website/build/debug
ERROR uwe::opts          > Plugin path is not a directory ../../plugins/std/core
```

I removed the `site.lock` and tried again, same error.

This is the what `site.toml` looks like

```
"std::core" = { path = "../../plugins/std/core", version = "~4.1.3", apply = { styles = ["**"] } }
```

Maybe relevant?

**Comment by tmpfs** [comment]
@chrism, sorry about that I was testing the plugin locally, just remove the `path` part so it looks like:

```toml
"std::core" = {version = "~4.1.3", apply = { styles = ["**"] } }
```

If you get other plugin errors you may need to wipe out `site.lock` again but hopefully it will work after that change.

**Comment by chrism** [comment]
changed it to

```
"std::core" = { version = "~4.1.3", apply = { styles = ["**"] } }
```

deleted `site.lock` but am getting this now

```
» uwe --live
 INFO  workspace::options > clean /Users/chrismasters/git/uwe-website/build/debug
 INFO  workspace::options > mkdir /Users/chrismasters/git/uwe-website/build/debug
 INFO  plugin::resolver   > Update registry cache
 INFO  scm::pull          > Pull origin/main in /Users/chrismasters/.uwe/runtime
 INFO  plugin::resolver   > Installing dependencies
 INFO  plugin::resolver   > Writing lock file /Users/chrismasters/git/uwe-website/site.lock
 INFO  workspace::project > Use std::link::github@1.0.2
 INFO  workspace::project > Use std::menu@1.0.0
 INFO  workspace::project > Use std::sidebar@1.0.1
 INFO  workspace::project > Use std::crumbtrail@1.0.1
 INFO  workspace::project > Use font::inter::regular@3.15.0
 INFO  workspace::project > Use std::core@4.1.10
 INFO  workspace::project > Use std::link@1.0.2
 INFO  workspace::project > Use std::listing@1.0.2
 INFO  workspace::project > Use font::inter@3.15.0
 INFO  workspace::project > Use std::note@1.0.0
 INFO  workspace::project > Syntax highlighting on
 ERROR uwe::opts          > Syntax error, path delimiter (.) not allowed here
```

**Comment by tmpfs** [comment]
Ah, that looks like i need to release a new version and you need an update. Will release an update first thing in the morning, apologies!

**Comment by chrism** [comment]
No problem no hurry!

**Comment by tmpfs** [comment]
@chrism, I have published `0.40.16` which I hope will fix this - let's hope that is the end of the snags!

If you are curious this was related to adding support for absolute paths to the `include` helper. Previously, the template engine treated any path starting with a path delimiter (`.` or `/`) as an error; now it accepts a leading slash to indicate an absolute path. 

The error you saw was because the source files for the website are attempting to use the new syntax but the executable you have installed does not yet support it.

Starting a path with a period is still considered an error. FYI, the correct syntax to get to a variable at the root of the template is to use `@root` when in a block scope:

```handlebars
{{@root.href}}
```

This mimics the handlebars behavior.

**Comment by tmpfs** [comment]
Closing this as fixed by `uwe task check-deps` implementation.
